README.md

```diff
@@ -196,7 +216,7 @@ cache:
   meta:
     ttl: 20
   transaction:
-    ttl: 20
+    ttl: 300
```

Comment:
I believe this TTL is in seconds. Is this documented?

Reply:
yup, it is documented elsewhere. I added in some comments to the yaml example to explicitly say it is in seconds for the readme.